alpha/declcfg/load.go

```diff
@@ -272,7 +268,7 @@ func extractCSV(objs []string) string {
 
 // LoadReader reads yaml or json from the passed in io.Reader and unmarshals it into a DeclarativeConfig struct.
 func LoadReader(r io.Reader) (*DeclarativeConfig, error) {
-	cfg := &DeclarativeConfig{}
+	cfg := &DeclarativeConfig{Deprecations: make(map[string]Deprecation)}
```

Comment:
I'm surprised you need to initialize the map here - why?

Reply:
If you don't, then the merge generates a panic assigning to nil map. 
We could catch it when we sense our first deprecations schema instance, but it seemed cleaner to init here.

Reply:
Your merge code already has:

```go
		if destination.Deprecations == nil {
			destination.Deprecations = make(map[string]Deprecation)
		}
```

Are you certain it will fail? :)

I think it's nicer to always handle the `nil`-map there instead of hoping that everyone initializes it correctly.

Reply:
Son of a ... I thought I took that out in favor of the init policy.  That's why this is a draft, man.  It's all belt-and-suspenders up in here. :)

Reply:
Have to init that map here or we can't create a struct with a non-nil map.  Merge would propagate it to the destination safely if that path hadn't previously encountered a deprecation instance.